src/server/db/models/SupplierBankAccount.js

```diff
@@ -0,0 +1,124 @@
+'use strict';
```

Comment:
We need syntax validation on server and client for well defined formats.
Eg Ivan and swift. For Iban also the checksum (pos 3+4,  after country code) need to be implemented...